lib/power_action_utils.pm

```diff
@@ -276,14 +276,21 @@ sub power_action {
             reboot_x11;
         }
         elsif ($action eq 'poweroff') {
-            if (is_backend_s390x) {
```

Comment:
I would suggest:
            if (is_backend_s390x) {
                if (check_var("DESKTOP", "gnome")) {
                    send_key "ctrl-alt-delete";
                    assert_screen 'logoutdialog', 15;
                    assert_and_click 'gnome-shell_shutdown_btn';
                    if (get_var("SHUTDOWN_NEEDS_AUTH")) {
                        assert_screen 'shutdown-auth';
                        type_password;
                        send_key "ret" && return 1;
                    }
                }
                else {
                    record_info('poo#114439', 'Temporary workaround, because shutdown module is marked as failed on s390x backend when shutting down from GUI.');
                    select_console 'root-console';
                    enter_cmd "$action";
            }
            else {
                poweroff_x11;
            }

Reply:
no, I'm afraid to say your suggestion is not correct
because all other cases are covered like:
  # Shutdown takes longer than 60 seconds on SLE12 SP4 and SLE 15
    if (is_sle('12+') && check_var('DESKTOP', 'gnome') && ($action eq 'poweroff')) {
        $soft_fail_data = {bugref => 'bsc#1055462', soft_timeout => 60, timeout => $shutdown_timeout *= 3};

 Please see my verification test run for zkvm provided above.

Reply:
hmm, However, I checked the VR 
https://openqa.suse.de/tests/9230121#step/shutdown/10, I got 
`poo#114439 Temporary workaround, because shutdown module is marked as failed on s390x backend when shutting down from GUI.

Do we need this workaround on s390x + zKVM? 

Anyway, please go ahead, actually, I am not familiar with this part, just wondering why you remove 'poweroff_x11' for all platforms.
`

Reply:
hmm, However, I checked the VR 
https://openqa.suse.de/tests/9230121#step/shutdown/10, I got 
`poo#114439 Temporary workaround, because shutdown module is marked as failed on s390x backend when shutting down from GUI.

Do we need this workaround on s390x + zKVM? 

Anyway, please go ahead, actually, I am not familiar with this part, just wondering why you remove 'poweroff_x11' for all platforms.

Reply:
I checked this again and I think now I can even to try fix both now, I wondered as well that record_info about 'poo#114439' refers to zkvm.  Actually the issue was only zVM, for for safety's sake, I just keep poweroff_x11 for the moment. I will check further for potenial issue.